### PR TITLE
Split core app to work with django-terra-utils

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
+django-terra-utils==0.3.0
```
Comment (nahuelange) [inline requirements.txt]: This package, and django-geostore shouldn't fixed to 0.3.x ?

File: setup.py [hunk 2]
```diff
@@ -29,16 +29,16 @@ def read(fname):
         "License :: OSI Approved :: MIT License",
     ],
     install_requires=[
+        "django-terra-utils==0.3.0",  # old terracommon.core
```
Comment (nahuelange) [inline setup.py]: Same here

File: requirements.txt (hunk 1 above)
Comment (submarcos) [reply requirements.txt]: done

File: setup.py (hunk 2 above)
Comment (submarcos) [reply setup.py]: done

**Pull request merged**